drivers/hmis/app/graphql/types/hmis_schema/ce_referral_step.rb

```diff
@@ -8,19 +8,32 @@
 
 module Types
   class HmisSchema::CeReferralStep < Types::BaseObject
-    field :id, ID, null: false
+    # object is a Hmis::WorkflowExecution::Step
+
+    field :id, ID, null: false # unique identifier for this step based on node and instance
+    field :step_id, ID, null: true # the DB identifier of this step, if it is persisted
```

Comment:
this is ok, a more concise option (instead of specifying `def step_id` down below) is to use `method`:
```suggestion
    field :step_id, ID, null: true, method: :id
```